### PR TITLE
Fix material for brass knuckles

### PR DESCRIPTION
**Pull request by cainiaowu**
They should be made of emm ... brass!

### CODE REVIEW

**Comment by Night-Pryanik** [comment]
Changing material to brass will make them non-repairable.

**Comment by cainiaowu** [comment]
I would also like to beable to repair brass items, but the problem is that we currently dont have brass ingot or something similar that could be used as repair_with.

File: data/json/items/tool_armor.json [hunk 1]
```diff
@@ -2041,7 +2041,7 @@
         "symbol" : "{",
         "description" : "A brass-plated alto saxophone with a neck strap, so you can wear it while playing.",
```
Comment (kevingranade) [inline data/json/items/tool_armor.json]: Adjust description as well, just removing "-plated" will do.

File: data/json/items/tools.json [hunk 2]
```diff
@@ -7803,7 +7803,7 @@
     "price": 7500,
```
Comment (kevingranade) [inline data/json/items/tools.json]: Remove "-plated" from this description too.

**Comment by kevingranade** [comment]
Brass repair would be welcome, it has a much lower melting point than iron.

**Pull request merged**